less/index.less

```diff
@@ -25,152 +92,284 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
 	display: block;
 }
+
 body {
 	line-height: 1;
 }
-ol, ul {
+
+ol,
+ul {
 	list-style: none;
 }
-blockquote, q {
+
+blockquote,
+q {
 	quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
+
+blockquote:after,
+blockquote:before,
+q:after,
+q:before {
 	content: '';
 	content: none;
 }
+
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
-
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+body,
+html {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 10px;
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 10px;
 }
 
 .container {
-    width: 880px;
-    margin: auto;
+	max-width: 880px;
+	margin: 0 auto;
+	padding: 0 20px;
+	@media (max-width: 768px) {
+		max-width: 768px;
+	}
+	@media (max-width: 400px) {
+		max-width: 400px;
+	}
 }
+// variables
+@nav-link-color: #b0b0b0;
+@nav-link-color-hover: black;
+@p-text-color: #555;
+@base-font-size: 16px; // mixins
 
-/* Header Styles */
-
-header {
-    width: 100%;
-    display: flex;
+.flex() {
```

Comment:
Maybe you're just getting some practice, but it's a little silly to make a mixin for this one line. I'll share one interesting idea another student employed:

>`//.flex-mix(@dir: column; @align: center); || .flex-mix(@just: center;) || etc`
>`.flex-mix(@just: flex-start; @align: stretch; @dir: row; @wrap: nowrap) {`

>>  `display: flex;`
>  `justify-content: @just;`
>  `align-items: @align;`
>  `flex-flow: @dir @wrap;`

>`}`

Sorry if it comes out weird, I'm having trouble getting github to style it correctly. Anyway, this is kind of a neat idea. I see in the code below that you used some mixins that make a lot of sense, but I'm showing you this to get you thinking about how you might employ some parametric mixins with some default settings as well. (So with this, you never have to give all or any values into the mixin.) I'm not saying this is the best application, but it shows you some possibilities.